.circleci/config.yml

```diff
@@ -1,318 +1,113 @@
-          filters: *integration-master_filters
-
-      - test:
-          name: install-ubuntu-versioned-master
-          executor: orb-tools/ubuntu
-          node-version: "8.9.4"
-          npm-version: "6.4.1"
-          yarn-version: "1.15.2"
-          filters: *integration-master_filters
-
-      # alpine
-      - test:
-          name: install-alpine-master
-          executor: orb-tools/alpine
-          filters: *integration-master_filters
-
-      - test:
-          name: install-alpine-versioned-master
-          executor: orb-tools/alpine
-          node-alpine-version: nodejs
-          npm-version: "6.4.1"
-          yarn-version: "1.15.2"
-          filters: *integration-master_filters
-
-      # oracle
-      - test:
-          name: install-oracle-master
-          executor: orb-tools/oracle
-          node-version: "8.9.4"
-          filters: *integration-master_filters
-
-      - test:
-          name: install-oracle-versioned-master
-          executor: orb-tools/oracle
-          npm-version: "6.4.1"
-          yarn-version: "1.15.2"
-          filters: *integration-master_filters
-
-      # patch, minor, or major publishing
-      - orb-tools/dev-promote-prod:
-          name: dev-promote-patch
-          context: orb-publishing
-          orb-name: circleci/node
-          ssh-fingerprints: 2a:2b:49:ea:64:7d:39:4a:58:17:18:17:b2:5c:f6:a8
-          cleanup-tags: true
-          requires: *prod-deploy_requires
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /master-patch.*/
-
-      - orb-tools/dev-promote-prod:
-          name: dev-promote-minor
-          context: orb-publishing
+      # your integration test jobs go here: essentially, run all your orb's
+      # jobs and commands to ensure they behave as expected. or, run other
+      # integration tests of your choosing
+      - integration-test-install-linux
+      - integration-test-install-mac
+      - node/test:
+          name: node-test-job
+          app-dir: "~/project/sample"
+      - orb-tools/dev-promote-prod-from-commit-subject:
           orb-name: circleci/node
-          ssh-fingerprints: 2a:2b:49:ea:64:7d:39:4a:58:17:18:17:b2:5c:f6:a8
-          cleanup-tags: true
-          release: minor
-          requires: *prod-deploy_requires
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /master-minor.*/
-
-      - orb-tools/dev-promote-prod:
-          name: dev-promote-major
           context: orb-publishing
-          orb-name: circleci/node
-          ssh-fingerprints: 2a:2b:49:ea:64:7d:39:4a:58:17:18:17:b2:5c:f6:a8
-          cleanup-tags: true
-          release: major
-          requires: *prod-deploy_requires
+          add-pr-comment: true
+          bot-token-variable: GHI_TOKEN
+          bot-user: cpe-bot
+          fail-if-semver-not-indicated: true
+          publish-version-tag: false
```

Comment:
Should we set `publish-version-tag` to `true` and retain `ssh-fingerprints`? I've been doing so for a few of the projects I converted to the osk pipeline recently.

Reply:
Enabled. Pushing momentarily.